src/typings.d.ts

```diff
@@ -31,3 +31,7 @@ interface AutocompletionOptions {
   path: string;
   size: number;
 }
+
+interface AppConfig {
```

Comment:
since this config is for editor, it might be better to name it here `EditorConfig`

Actually I usually add it next to declaration when it has only single property like

`@Input config: {schemaOptions?:  Object};`